srcpkgs/apcupsd/files/apcupsd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -d /run/apcupsd ] || mkdir /run/apcupsd
```

Comment:
probably want -p

Reply:
if the item is a file and not a dir, mkdir is failing even with -p.
i also assume /run always exist, else system is probably broken…